zeebe/atomix/cluster/src/test/java/io/atomix/raft/RaftCorruptedDataTest.java

```diff
@@ -59,6 +62,10 @@ public void upToDateFollowerShouldNotLoseDataWhenQuorumExperienceCorruption() th
     // wait for the two corrupted nodes to form quorum
     Awaitility.await("corrupted nodes form a quorum")
         .until(() -> server0.isLeader() || server1.isLeader());
+    raftRule.appendEntries(1000);
+    raftRule.takeCompactingSnapshot(server0, 500, 1);
+    raftRule.takeCompactingSnapshot(server1, 500, 1);
+
     // when
```

Comment:
❓ Wondering whether we maybe should make a separate regression test for this?

Reply:
:x: Agreed, we should have this as two different test cases.